content/en/_index.md

```diff
@@ -1,17 +1,14 @@
 ---
 title: OpenTelemetry
 description: >-
-  High-quality, ubiquitous, and portable telemetry to enable effective
-  observability
+  কার্যকর অবজারভেবিলিটির জন্য উচ্চ-মানের, সর্বজনীন ও পোর্টেবল টেলিমেট্রি প্রদান
 outputs:
   - HTML
-  # Include the following for `content/en` ONLY
-  - REDIRECTS
-  - RSS
```

Comment:
Instead of removing them, consider commenting them out. That way, we can still track these items for future reference.

 ```
 # Include the following for `content/en` ONLY
 #- REDIRECTS
 #- RSS
```